library/src/main/kotlin/de/adesso/junitinsights/model/EventLog.kt

```diff
@@ -1,21 +1,9 @@
 package de.adesso.junitinsights.model
 
-import com.google.gson.Gson
-import de.adesso.junitinsights.tools.IReportCreator
 import de.adesso.junitinsights.tools.InsightProperties
-import de.adesso.junitinsights.tools.ReportCreator
-import org.slf4j.LoggerFactory
-import org.springframework.core.io.ClassPathResource
-import java.io.File
-import java.io.InputStreamReader
-import java.io.PrintWriter
-import java.time.LocalDateTime
-import java.time.format.DateTimeFormatter
 
 object EventLog {
-    private var events: ArrayList<Event> = ArrayList()
-    private var currentDate: LocalDateTime = LocalDateTime.now()
-    var reportCreator: IReportCreator = ReportCreator
+    internal var events: ArrayList<Event> = ArrayList()
```

Comment:
Is it really good practice to change the visibility of attributes just for writing tests? In my understanding the tests should be completely independent of the implementation.

Reply:
True, and I really wasn't happy with that change either. It's not required for testing though: I wanted to be able to point a `ReportCreator` to the list of events from the `EventLog` without needing the `ReportCreator` to know about the `EventLog` (and only about a Java List of Events).

Would be grateful for a better idea to achieve this.